zaza/openstack/charm_tests/keystone/tests.py

```diff
@@ -21,7 +21,7 @@
 
 import zaza.model
 import zaza.openstack.utilities.exceptions as zaza_exceptions
-import zaza.openstack.utilities.juju as juju_utils
+import zaza.utilities.juju as juju_utils
```

Comment:
This removes deprecation warnings after https://github.com/openstack-charmers/zaza/pull/368